environment.yml

```diff
@@ -3,37 +3,45 @@ name: magic-lst
 channels:
   - conda-forge
 dependencies:
-  - python=3.8
+  - python=3.11
   - pip
+  - numpy
+  - astropy=5
   - black
   - nbsphinx
-  - numpy=1.21
-  - ctapipe=0.12
-  - gammapy=0.19.0
-  - cython
+  - ctapipe=0.19.2
+  - gammapy=1.1
+  - ctaplot~=0.6.4
+  - h5py
+  - ipython
+  - jupyter
+  - matplotlib=3.7
+  - notebook
   - graphviz
   - iminuit>=2
-  - joblib
+  - joblib~=1.2.0
   - numpydoc
   - jupyterlab=3.5.0
-  - pandas
+  - pandas>=2
   - pre-commit
-  - protozfits=2.0
+  - toml
+  - protobuf=3.20
+  - pydata-sphinx-theme
+  - pyparsing
   - psutil
   - pytest
   - pytest-cov
   - pytest-runner
   - pyyaml
-  - scikit-learn
+  - scikit-learn=1.2
   - sphinx
   - sphinx-automodapi
   - sphinx-design
   - sphinx_rtd_theme
-  - pydata-sphinx-theme
-  - uproot~=4.1
-  - corsikaio
-  - pip:
-      - lstchain~=0.9.6
-      - ctapipe_io_magic~=0.4.7
-      - ctaplot~=0.5.5
-      - pyirf~=0.6.0
+  - numba>=0.56
+  - uproot
+  - pymongo
+  - jinja2 # pinned for bokeh 1.0 compatibility
```

Comment:
similar here, setup uses ~3.0.2, while here there is no version specified, despite the comment says that it is "pinned"

Reply:
I think we should not use 'fixed' versions unless necessary to create a working environment because versions depends on the versions of the other dependencies and we could run into clashes when we update the environment/when new versions are released. The comment is old, we should check again the compatibility (btw, conda search for compatible versions, if not fixed, when creating the environment).  @aleberti: what about this?

Reply:
I think it is ok to have jinja~=3.0.2, because in any case it is the one ctapipe has, so we need to be compatible in this sense.